rails-commenting.rb

```diff
@@ -6,25 +6,43 @@
 
 # FILE: app/controller/blog_posts_controller.rb
 
-# ---1)
+# ---1) The controller is the brains of the operation. It is in charge of taking in the request from the user and is in charge of the coordination between the model and the views. It is either created by using generate controler or generate resource. It is also in charge of caching, helper modules and sessions. Helpers, do lots of stuff, one example is with the view, it eliminates the need for having an "if statement" in the view, ex 
+# view:
+# <% if @user && @user.email.present? %>
+# <%= @user.email %>
+# <% end %>
+# instead you can put it in the controller ex
+# controller:
+# module SiteHelper
+  # def user_email(user)
+    # user.email if user && user.email.persent?
+  # end
+# end    
+# for more info see https://mixandgo.com/learn/ruby-on-rails/beginner-guide-rails-helpers
 class BlogPostsController < ApplicationController
```

Comment:
What relationship does < show between BlogPostsController and ApplicationController?